packages/meditrak-server/src/routes/index.js

```diff
@@ -167,6 +168,7 @@ export default {
   getDashboards: useRouteHandler(GETDashboards),
   getDashboardItems: useRouteHandler(GETDashboardItems),
   getDashboardRelations: useRouteHandler(GETDashboardRelations),
+  getDashboardVisualisations: useRouteHandler(GETDashboardVisualisation),
```

Comment:
It's minor but any reason for the difference in plurality? Shouldn't GETDashboardVisualisation have an "s"?

Reply:
And the filename would also be plural right?

Reply:
Ah yes, this was actually intentional because I don't think we will ever get multiple dashboard visualisations. But I understand the concern of inconsistency. Have changed it to plural